src/modules/site-v2/base/views/tools/heritability_calculator.py

```diff
@@ -276,18 +274,14 @@ def report(id):
 
     'ready': ready,
 
-    # TODO: The HTML file expects a variable called "fnam" -- is that a typo?
     'fname': datetime.today().strftime('%Y%m%d.') + trait,
 
     'hr': job.report,
     'data': data,
     'result': result,
+    'error': job.get_error(),
```

Comment:
`JobPipeline` object provides access to the error, so we don't have to use the `PipelineOperation` (can instead let it be handled implicitly by the `Runner`)